tests/suites/main_test.function

```diff
@@ -281,6 +281,14 @@ int main(int argc, const char *argv[])
 #if defined(MBEDTLS_MEMORY_BUFFER_ALLOC_C) && \
     !defined(TEST_SUITE_MEMORY_BUFFER_ALLOC)
     unsigned char alloc_buf[1000000];
+#endif 
+    if( platform_setup() != 0 )
+    {
+        mbedtls_fprintf( stderr, "FATAL: Failed to initialize platform" );
+        return -1;
```

Comment:
Mbed TLS coding standard wise it should be ```return( -1 )```.

Reply:
Done, despite the fact that in many places in this file it's not obeyed.